frontend/src/components/ShootDetails/GShootLifecycleCard.vue

```diff
@@ -141,14 +138,38 @@ SPDX-License-Identifier: Apache-2.0
           </template>
           <g-list-item-content>
             Delete Cluster
+            <template
+              v-if="canForceDeleteShoot"
+              #description
+            >
+              <div class="d-flex">
+                <v-icon
```

Comment:
![Screenshot 2024-01-25 at 11 04 37](https://github.com/gardener/dashboard/assets/5526658/097f3977-d81b-469b-bb60-54acd2323c35)
I think we should do it similar like for the last maintenance status to be more consistent